RELEASE.md

```diff
@@ -5,7 +5,7 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
-# Upcoming Release
+# Release 9.0.0
```

Comment:
Upgrade to Node 18 (#1848) not #1811

Reply:
and 'Add support for `JSONDatast` preview should be moved to major features and improvements, i think

Reply:
can we also add #1819, #1798,

Reply:
Sure, Let me add that.